garage/src/hooks/useTablelandConnection.ts

```diff
@@ -1,7 +1,7 @@
-import { Database, Validator } from "@tableland/sdk";
+import { Database, Validator, helpers } from "@tableland/sdk";
 import { chain } from "../env";
 
-const db = Database.readOnly(chain.id);
+const db = new Database({ baseUrl: helpers.getBaseUrl(chain.id) });
```

Comment:
iirc `readonly` was a bit confusing because you'd specify a single chain...but reads can be cross chain. so, that was dropped and just keeps it generalized